catroid/src/org/catrobat/catroid/web/ServerCalls.java

```diff
@@ -152,7 +151,7 @@
 
 	private static final ServerCalls INSTANCE = new ServerCalls();
 
-	public static boolean useTestUrl = true;
+	public static boolean useTestUrl = false;
```

Comment:
https://github.com/Catrobat/Catroid/pull/1797/files?diff=unified#diff-8e10fb43a2b53e349b11d6c768b1a65fL153
Other code also relies on this flag. I don't know if false or true is the correct value, so I changed it to highlight it. :wink:
@LauraBebek, please take a look at this.